reddit/reaper.go

```diff
@@ -62,6 +63,7 @@ func newReaper(c reaperConfig) reaper {
 
 func (r *reaperImpl) reap(path string, values map[string]string) (Harvest, error) {
 	r.rateBlock()
+	r.url(r.path(path, r.reapSuffix), values)
```

Comment:
I don't think this function has side effects and its result is not stored anywhere.

Reply:
Wops, debug remains :)